src/main/java/uk/gov/digital/ho/hocs/casework/startup/RefreshBankHolidays.java

```diff
@@ -0,0 +1,24 @@
+package uk.gov.digital.ho.hocs.casework.startup;
+
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.context.event.ApplicationReadyEvent;
+import org.springframework.context.event.EventListener;
+import org.springframework.stereotype.Service;
+import uk.gov.digital.ho.hocs.casework.api.BankHolidayService;
+
+@Service
+@Slf4j
+public class RefreshBankHolidays {
```

Comment:
I don't think we want this, they won't change multiple times a day - we have three instances in prod at any one time and we'll be fetching data for a year at a time and more?

Reply:
I have now addressed this